src/main/java/pl/edu/zut/mad/schedule/model/outer/Day.java

```diff
@@ -0,0 +1,17 @@
+package pl.edu.zut.mad.schedule.model.outer;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+public class Day {
+
+    @JsonFormat(pattern = "yyyy-MM-dd")
```

Comment:
Finally I think about date format and in my opinion it should be `dd-MM-yyyy`. It looks more friendly and there will be no conflicts on client side.